### PR TITLE
First contribution

### PR DESCRIPTION
**Pull request by danishhansari**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.18.3@sha256:a452d6273ad03a47c2f29b898d6bb57630e77baf839651ef77d03e4e049c5bf3 AS builder
```
Comment (Hayden-IO) [inline Dockerfile:16]: You'll need to sync from HEAD as this commit was already added to main.

**Comment by Hayden-IO** [comment]
For PRs, it's helpful to write an informative title, like "Grammatical updates to README" and a description of the change.

**Comment by ansari-danish** [comment]
> For PRs, it's helpful to write an informative title, like "Grammatical updates to README" and a description of the change.

I forgotten this time but Next time i will those stufff

**Comment by danishhansari** [comment]
> For PRs, it's helpful to write an informative title, like "Grammatical updates to README" and a description of the change.

I am really sorry this time but next time i do 🥺🥺❤️

File: Dockerfile (hunk 1 above)
Comment (ansari-danish) [reply Dockerfile:16]: > > For PRs, it's helpful to write an informative title, like "Grammatical updates to README" and a description of the change.
> 
> I am really sorry this time but next time i do 🥺🥺❤️



> You'll need to sync from HEAD as this commit was already added to main.
I Messed up a bit so i forgot to write a description

**Comment by danishhansari** [comment]
> For PRs, it's helpful to write an informative title, like "Grammatical updates to README" and a description of the change.

I messed up a bit so i forgot

**Comment by danishhansari** [comment]
hey can u please review my PR

**Review by dlorenc — APPROVED** [review]

**Review by dlorenc — CHANGES_REQUESTED** [review]
The Dockerfile change needs to get dropped here before we can merge.

**Comment by Hayden-IO** [comment]
Also bumping the comment about updating the PR title, you should be able to edit it.

**Comment by danishhansari** [comment]
> The Dockerfile change needs to get dropped here before we can merge.

I didn't make any Changes in Dockerfile i only change Readme file

**Comment by Hayden-IO** [comment]
In the PR, there are two commit hashes. The first is `743f9c7`, which shows you modifying the Dockerfile. You need to rebase against `main` to drop this.

**Pull request closed without merging**